src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,33 @@
+.cards {
+  --columns-count: 1;
+  display: grid;
+  column-gap: 48px;
+  row-gap: 46px;
+  grid-template-columns: repeat(var(--columns-count), 200px);
+  justify-items: center;
+  justify-content: center;
+
+  max-width: max-width(--columns-count, 200px, 48px);
```

Comment:
I've checked and this property doesn't work in browser, so maybe you don't need it